x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/index.test.tsx

```diff
@@ -48,18 +48,45 @@ describe('when on the hosts page', () => {
     describe('when list data loads', () => {
       beforeEach(() => {
         reactTestingLibrary.act(() => {
+          const hostListData = mockHostResultList({ total: 3 });
+          [HostStatus.ERROR, HostStatus.ONLINE, HostStatus.OFFLINE].forEach((status, index) => {
+            // @ts-ignore
+            hostListData.hosts[index].host_status = status;
```

Comment:
Could you add a comment explaining to the ts ignore? I  think you can tack it on
```
// ts-ignore Because some reason.
```

Also, why is the ts ignore needed?

Reply:
Sure. I found that the `mockHostResultsList()` - which is suppose to represent the API response - is coded as Immutable. For these test cases, all I need is to validate different statuses and would like to mutate the value prior to dispatching the action - thus instead of trying to reconstruct the entire message, I just `ts-ignore`'d it.

Personally, I don't think that should be the case 😞 . Only the `payload` of the Store action should be immutable. I will rework this to try and avoid the `ts-ignore`